### DISCUSSION TITLE
How to backup Gitbook content

### DISCUSSION THREAD

**Discussion by wolfijenne** [GitbookIO/community#505] [1. How to] [answered]
We are using Gitbook as our business process documentation tool and love it. How can we store a backup somewhere else for the worst case scenarios?

**Comment by ahcsco** [comment, marked as answer]
Sync it to Github and you'll have the backup in .md files.

**Reply by thebiglabasky** [reply]
Exactly. We otherwise don't offer backup for settings or else. Content can be synced to a GitHub or GitLab repo for backup purposes
